.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!--
+When filing an issue, please fill in the requested fields as thoroughly as you can and then delete this comment
+-->
+
+* Description of issue
+
+    description
+
+* `python -c "import sys; print(sys.version)"` output
+
+    ```
+    replace with output
+    ```
+
+* `python -m pip list` output
+
+    ```
+    replace with output
+    ```
+
+* Diver and version (this is the installed driver version, not the Python API version)
```

Comment:
> Diver and version (this is the installed driver version, not the Python API version)

`Driver runtime version (this is the installed version, not the Python API version)`

A python one-liner to print that would be good. Or instructions on getting that from MAX in case customer didn't make it that far.